src/main/java/Event.java

```diff
@@ -0,0 +1,12 @@
+public class Event extends Task{
+    private String eventTime;
```

Comment:
good use of camelCase, would highly recommend using camelCase throughout the project